src/main/java/seedu/mtracker/model/Instrument.java

```diff
@@ -1,30 +1,42 @@
 package seedu.mtracker.model;
 
+import java.util.HashMap;
+import java.util.HashSet;
+
 public abstract class Instrument {
     protected String name;
     protected double currentPrice;
     protected String sentiment;
     protected boolean isDone;
 
-    protected static final String EMPTY_STRING = "";
-    protected static final String FILE_SEPARATOR = ";";
+    protected static HashSet<String> validAttribute;
+
     public static final String SEMICOLON_SEP = "; ";

```

Comment:
Sry it wasnt caught in the other prs, but can rearrange. Public -> protected -> private